plugin/storage/es/spanstore/reader.go

```diff
@@ -459,6 +468,10 @@ func (s *SpanReader) buildServiceNameQuery(serviceName string) elastic.Query {
 	return elastic.NewMatchQuery(serviceNameField, serviceName)
 }
 
+func (s *SpanReader) buildAnyServiceNameQuery() elastic.Query {
+	return elastic.NewWildcardQuery(serviceNameField, defaultAnyServiceName)
```

Comment:
I suggest removing `defaultAnyServiceName` constant and using `*` explicitly here.